src/operator/tensor/broadcast_reduce_op.h

```diff
@@ -1580,6 +1581,20 @@ void PickOpBackward(const nnvm::NodeAttrs& attrs,
   });
 }
 
+inline std::string get_reduce_axes_description(const std::string& op_name, int line) {
```

Comment:
why do you want to inline this?

Reply:
The short answer is that the function was formerly in broadcast_reduce_op_value.cc (with the inline keyword) and I copied it intact to broadcast_reduce_op.h.

Also, removing `inline` breaks the compile with a 'multiple definitions' error.  One shouldn't put the definition of a stand-alone function in a header file that's part of multiple translation units without the inline keyword.

Reply:
Of course I know that. The question is why isn't declared on the header and defined in the implementation file as usual. Seems like a big function to inline. Which adds bloat.